pkg/kubelet/cm/qos_container_manager_linux.go

```diff
@@ -139,7 +139,12 @@ func (m *qosContainerManagerImpl) Start(getNodeAllocatable func() v1.ResourceLis
 }
 
 func (m *qosContainerManagerImpl) setCPUCgroupConfig(configs map[v1.PodQOSClass]*CgroupConfig) error {
-	pods := m.activePods()
+	var pods []*v1.Pod
```

Comment:
for example, if there is a code block that has a nil active pods func, can we instead pass in a no-op version that just returns an empty list?